Attempt at Solving Issue #443

questionable-masquarade opened the pull request:
## Description of changes
Modifies cyclerate (every cycle it runs through a list and modifies the seed in the tray.) Modifies the chance for certain events in the tray. Hopefully balances it out for hydroponics on a longer timeframe. Will adjust as needed. 

## Why and what will this PR improve
Makes hydroponics take longer to hopefully make it work on the persistent timeframe. Hopefully satisfies #443 

## Authorship
All me bay bee.

## Changelog
:cl:
tweak: Hydroponics rates
/:cl:

questionable-masquarade committed:
Update tray_process.dm

Lowers the chance per cycle for certain events. Hoping to solve #443

questionable-masquarade committed:
Update tray.dm

Makes the cycles longer to hopefully make hydroponics something that takes longer.

genessee-forgot-his-password-again reviewed (changes requested):
As I mentioned in our discussions on Discord, you cannot make changes outside of the persistence modpack.

questionable-masquarade committed:
rebalances some of the things I overestimated

questionable-masquarade committed:
Update seed_datums.dm

messes with maturation and production numbers so that with the reduced cycle rates we can have more fair hydroponics

questionable-masquarade commented:
> As I mentioned in our discussions on Discord, you cannot make changes outside of the persistence modpack.

I have talked to Natakilar about it and it should be fine.

questionable-masquarade commented:
do not merge. have to go through seed datums again to balance it further.

questionable-masquarade committed:
balances seeds even more

makes things fit a time frame and puts seeds into tiers
tier 0 = aiming for 15 minutes
tier 1 = aiming for 30 minutes + more if its a better tier 1
tier 2 = aiming for an hour + more
tier 3 = aiming for an hour + other nerfs

questionable-masquarade commented:
hopefully this makes it ready to merge. requesting review + comments

PsyCommando commented on code/modules/hydroponics/seed_datums.dm:
```diff
@@ -8,9 +8,9 @@
+	set_trait(TRAIT_MATURATION,1)
+	set_trait(TRAIT_PRODUCTION,2)
+	set_trait(TRAIT_YIELD,4)
 	set_trait(TRAIT_POTENCY,10)
```
Not sure why you're changing the production and maturation traits on those plants?

questionable-masquarade replied:
Hi, the production and maturation traits are changed to fit the new probabilities and cycle rates. 1 plant age averages out to 15 minutes, but as it is a probability, this can vary. Essentially, it's just balancing out.

PsyCommando replied:
Alright, just curious,
